### DISCUSSION TITLE
Create ADF

### DISCUSSION THREAD

**Discussion by aZtOcKdOg** [vAmigaWeb/vAmigaWeb#169] [Ideas]
It would be really great to be able to create a new ADF on the fly.  For eg. insert DF1: and save data to it (and save to local device storage using the existing save ADF feature in 'savestate')

**Comment by mithrendal** [comment]
Absolutely makes sense … coding is the easy part … I have to think about the UI i.e. where to add the insert DF1 button …

**Reply by aZtOcKdOg** [reply]
Hey great!

It seems on first insert (ADF or HDF) it prompts which 'slot' to put the disk into. But after boot and you insert say another ADF it no longer brings up that initial prompt for which device to insert into (ie DF0, DF1 etc)

So that popup is there just be great for that to come up every time anything is inserted?
